src/Microsoft.ML.Data/Prediction/Calibrator.cs

```diff
@@ -284,7 +284,7 @@ private protected static TCalibrator GetCalibrator(IHostEnvironment env, ModelLo
         }
     }
 
-    internal abstract class ValueMapperCalibratedModelParametersBase<TSubModel, TCalibrator> :
+    public abstract class ValueMapperCalibratedModelParametersBase<TSubModel, TCalibrator> :
```

Comment:
This and changes below expose some very complicated classes to the users. From my understanding, the major thing to be exposed in OVA is `TModel[]` and `CalibratedModel<TModel,TCalibrator>[]`. Is it possible to keep those classes here internal?

Reply:
Good point. I have changed it back to internal and made everywhere public use the `CalibratedModelParametersBase`